tests/api.c

```diff
@@ -55480,6 +55480,123 @@ static void test_wolfSSL_FIPS_mode(void)
 #endif
 }
 
+#ifdef WOLFSSL_DTLS
+/* The following is a copy of functions from internal.c that are normally
+ * and not in the public API. This is testing those functions, well, a
+ * copy of them. */
```

Comment:
Yes, keeping a copy in sync is not ideal. Putting function in `internal.h` isn't ideal either, as it will increase the size of every object it is included in, even if the functions aren't used. 

IMO we can just drop the static keyword and export them as internal API. These functions aren't short and I don't think the inlining change too much the performance.

Reply:
I had to make the function WOLFSSL_API, but since it is in internal.h I'm fine with that. I dropped _DtlsUpdateWindow is now wolfSSL_DtlsUpdateWindow.

Reply:
nice, just a question (in the review as well), why we can't use WOLFSSL_LOCAL instead of WOLFSSL_API?